src/the-2021-scheduler.md

```diff
@@ -173,8 +168,8 @@ if (states.good.normal?.review) {
 ```
 
 Because this is implemented in JavaScript, it is not limited to the computer
-version. AnkiMobile and AnkiDroid both support it as well, and AnkiWeb may
-support it in the future too. This will allow advanced users to make 
+version. AnkiMobile and AnkiDroid both support it as well, and AnkiWeb may also
+support it in the future. This will allow advanced users to make 
 adjustments to the standard scheduling behaviour, that apply on all platforms.
 
-The various scheduling states are described in SchedulingStates [here](https://github.com/ankitects/anki/blob/main/proto/anki/scheduler.proto).
+The various scheduling states are described in [Scheduling States](https://github.com/ankitects/anki/blob/main/proto/anki/scheduler.proto).
```

Comment:
```suggestion
The various scheduling states are described in [SchedulingStates](https://github.com/ankitects/anki/blob/main/proto/anki/scheduler.proto).
```
That's what it's called in the code -- no space.

Reply:
Typo on my part.